WireAPI/Tests/WireAPITests/APIs/ConversationsAPI/__Snapshots__/ConversationsAPITests/testGetConversations_givenAllAPIVersions_thenVerifyRequests.v1.txt

```diff
@@ -0,0 +1,4 @@
+- HTTPRequest
+  - path: "/conversations/list/v2"
```

Comment:
I believe the path here should be `/v1/conversations/list/v2`

Reply:
Right, thanks! f116a8049a2397553417de483d31cf6af551fe04